administration/serializers.py

```diff
@@ -0,0 +1,31 @@
+from djoser.serializers import UserSerializer
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework.serializers import SerializerMethodField
+
+User = get_user_model()
+
+
+class AdminUserSerializer(UserSerializer):
+
+    phone_number = SerializerMethodField('get_phone_number')
```

Comment:
`'get_phone_number'` string is redundand and should be removed (as it matches the name expected for the method by default).